server/opts_test.go

```diff
@@ -2826,3 +2826,66 @@ func TestDefaultAuthTimeout(t *testing.T) {
 		t.Fatalf("Expected auth timeout to be %v, got %v", 5, sopts.AuthTimeout)
 	}
 }
+
+func TestQueuePermissions(t *testing.T) {
+	cfgFmt := `
+		listen: 127.0.0.1:-1
+		no_auth_user: u
+		authorization {
+			users [{ user: u, password: pwd, permissions: {
+						sub: { %s: ["foo.> *.dev"] }
```

Comment:
Seem that there is some tabs/spaces issues here and next line?

Reply:
done